src/browser/components/MainPage.jsx

```diff
@@ -156,10 +156,24 @@ export default class MainPage extends React.Component {
       this.handleSelect(key);
     });
     ipcRenderer.on('select-next-tab', () => {
-      this.handleSelect(this.state.key + 1);
+      const currentOrder = this.props.teams[this.state.key].order;
+      const nextIndex = this.props.teams.map((team, index) => {
+        return {
+          index,
+          order: (team.order - currentOrder) - 1,
+        };
+      }).find((team) => team.order === 0 || Math.abs(team.order) === this.props.teams.length);
```

Comment:
I wonder if it wouldn't be easier if we calculate the next one and then just look for it.
```js
const nextOrder = (currentOrder + 1) % this.props.teams.length;
const nextIndex = this.props.teams.find((team) => team.order === nextOrder;
```

Reply:
Order isn't 0 based, so this doesn't work.

Reply:
just checked and it is 0 based, if you think we are assuming it is not at some other line of code, we should revisit that part for possible bugs.